plugins/out_stackdriver/stackdriver.c

```diff
@@ -2249,6 +2263,15 @@ static flb_sds_t stackdriver_format(struct flb_stackdriver *ctx,
             flb_sds_destroy(trace);
         }
 
+        /* Add spanId field into the log entry */
+        if (span_id_extracted == FLB_TRUE) {
+            msgpack_pack_str(&mp_pck, 6);
```

Comment:
Sorry I should have been more clear in my original comment; this one can also be changed to:
```
msgpack_pack_str_with_body(&mp_pck, "spanId", 6);
```